traveler-ios-ui/TravelerKitUI/UIUtilities/HeartButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  HeartButton.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-09-16.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+
+@IBDesignable class HeartButton: UIButton {
+    @IBInspectable var isFilled: Bool = false
```

Comment:
it might be better to piggy back off of `isSelected` for this. That way the VC's can not know of HeartButton and only the interface builder knows

Reply:
did we make this change?